main.go

```diff
@@ -0,0 +1,215 @@
+package main
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"regexp"
+	"time"
+
+	"github.com/howeyc/fsnotify"
+	_ "github.com/lib/pq"
+)
+
+var (
+	// example: 2014-10-31-15-15.json
+	filenameRegex     = regexp.MustCompile(`(\d{4}(-\d{2}){4})\.json$`)
+	datetimeRegex     = regexp.MustCompile(`([\d-]*)`)
+	datetimeFormat    = "2006-01-02-15-04"
```

Comment:
adding a clarifying comment but it's how Golang's `time` package manages formats. A format is made by declaring a specific date in the desired format.

reference date: `Mon Jan 2 15:04:05 MST 2006`
If I just wanted the minute, I'd use `04`. The hour: `15`. The hour in 12-hour: `03`. The year: `2006`, etc.

[docs](http://golang.org/pkg/time/#pkg-constants)